filters/annoyances.txt

```diff
@@ -6696,3 +6696,7 @@ the-masters-voice.com##*::selection:style(background-color: #338FFF !important)
 
 ! https://github.com/uBlockOrigin/uAssets/issues/14420
 hemas.pl##+js(aeld, contextmenu)
+
+! https://github.com/uBlockOrigin/uAssets/issues/14529
+lover92.net,lover93.net,lover94.net,lover95.net,lover96.net,lover97.net,lover98.net,lover99.net,lover100.net,lover101.net##+js(acis, history.pushState, history.go(1))
```

Comment:
https://github.com/uBlockOrigin/uAssets/commit/440b17fc147c1df2132b6cc71fc114f2ffc9fa71

Reply:
@JobcenterTycoon I think we should address in annoyance

Reply:
Similar issues like https://github.com/uBlockOrigin/uAssets/issues/8592 got fixed in `uBlock filters` too.

Its not just a annoyance like a blocked right click, it is malicious behavior which need to be blocked by `uBlock filters`